tests/topics/test_topic_list_endpoint.py

```diff
@@ -0,0 +1,175 @@
+"""Testing TopicList endpoint."""
+
+import jwt
+from sqlalchemy import desc
+
+from src.config import Config
+from src.topics.models import Topic
+
+
+def test_get_topic_list_endpoint_with_no_args_returns_whole_list(client, db_with_three_users_and_three_topics):
+    session = db_with_three_users_and_three_topics
+    topic_list = session.query(Topic).all()
+    authorization_token = jwt.encode({"user_id": 1}, Config.SECRET_KEY, algorithm="HS256")
+
+    result = client.get("api/topics", headers={"Authorization": f"Bearer {authorization_token}"})
+
+    assert result.status_code == 200
+    assert result.json == [{
+        "id": topic.id,
+        "author_id": topic.author_id,
+        "created_at": str(topic.created_at),
+        "description": topic.description,
+        "title": topic.title,
+    } for topic in topic_list]
+
+
+def test_get_topic_list_endpoint_with_order_by_returns_200(client, db_with_three_users_and_three_topics):
+    session = db_with_three_users_and_three_topics
+    authorization_token = jwt.encode({"user_id": 1}, Config.SECRET_KEY, algorithm="HS256")
+    ordered_topic_list = session.query(Topic).order_by(desc("title")).all()
+
+    result = client.get("api/topics", headers={"Authorization": f"Bearer {authorization_token}"},
+                        json={"order_by": "title,desc"})
```

Comment:
`order_by` is supposed to be query parameter, not in json-body. Yes, flask-restx allows us to provide it in any place and it will parse it anyway, but that's the flaw of the framework and actually it's not a good idea.

So let's provide query parameters as they supposed to be provided, no matter what flask_restx is able to do and where it can parameters from.

Please double check that for all topic-related tests